packages/backend/src/controllers/v2/FeatureFlagController.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiErrorPayload, FeatureFlag } from '@lightdash/common';
+import {
+    Get,
+    OperationId,
+    Path,
+    Request,
+    Response,
+    Route,
+    SuccessResponse,
+    Tags,
+} from '@tsoa/runtime';
+import express from 'express';
+import { BaseController } from '../baseController';
+
+@Route('/api/v2/feature-flag')
+@Response<ApiErrorPayload>('default', 'Error')
+@Tags('v2', 'Feature Flag')
+export class FeatureFlagController extends BaseController {
+    /**
+     * Get feature flag
+     */
+    @SuccessResponse('200', 'Success')
+    @Get('/{featureFlagId}')
+    @OperationId('Get feature flag')
+    async getFeatureFlag(
+        @Request() req: express.Request,
+        @Path() featureFlagId: string,
```

Comment:
Note that I typed featureFlagId as a string instead of FeatureFlag enum so we can easily extend commercial flags.